code/datums/controllers/job_controls.dm

```diff
@@ -13,12 +13,16 @@ var/datum/job_controller/job_controls
 	New()
 		..()
 		if (world.load_intra_round_value("solarium_complete") == 1 || derelict_mode)
-			src.staple_jobs = list(new /datum/job/command/captain/derelict {limit = 1;name = "NT-SO Commander";} (),
-			new /datum/job/command/head_of_security/derelict {limit = 1; name = "NT-SO Special Operative";} (),
-			new /datum/job/command/chief_engineer/derelict {limit = 1; name = "Salvage Chief";} (),
-			new /datum/job/security/security_officer/derelict {limit = 6; name = "NT-SO Officer";} (),
-			new /datum/job/research/medical_doctor/derelict {limit = 8; name = "Salvage Medic";} (),
-			new /datum/job/engineering/engineer/derelict {limit = 10; name = "Salvage Engineer";} (),
+			src.staple_jobs = list(new /datum/job/command/captain/derelict {limit = 1; name = "Nanotrasen Emergency Response Squad Leader";} (),
+			new /datum/job/command/head_of_personnel/derelict {limit = 1; name = "Nanotrasen Emergency Response Operative";} (),
+			new /datum/job/command/chief_engineer/derelict {limit = 1; name = "Nanotrasen Emergency Response Technician";} (),
+			new /datum/job/command/medical_director/derelict {limit = 1; name = "Nanotrasen Emergency Response Medic";} (),
```

Comment:
I'm not a huge fan of cutting the number of slots available for special roles so heavily. Going from ~25 slots of modified sec/eng/med to 4 slots of modified NTSC doesn't feel like an upgrade to me

Reply:
Having only 4 players with gear, with the rest basically needing to twiddling thier thumbs doesn't sound great at all.

I'd much rather the enemies get harder then the gear being limited to 4 sets